.github/dependabot.yml

```
@@ -8,3 +8,8 @@ updates:
       github-actions:
         patterns:
           - '*'
+
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

```suggestion
      interval: "monthly"
    groups:
      npm:
        patterns:
          - '*'
```

Let's add a group for this too so we don't get a million PRs

---

Sorry I missed this. Agreed. Thanks Jaime! 🙏